### PR TITLE
Update education-settings-configure-ios.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed the intune URL part.

### CODE REVIEW

**Comment by PRMerger12** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2752

File: memdocs/intune/fundamentals/education-settings-configure-ios.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ You can import information into SDS by using one of the following methods:
 
 ### Configure general settings
 
-1. Sign in to [Intune](https://go.microsoft.com/fwlink/?linkid=2090973).
+1. Sign in to [Intune]([https://go.microsoft.com/fwlink/?linkid=2090973](https://endpoint.microsoft.com)).
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/education-settings-configure-ios.md:83]:
```suggestion
1. Sign in to [Intune](https://endpoint.microsoft.com)).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger19** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/fundamentals/education-settings-configure-ios.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ You can import information into SDS by using one of the following methods:
 
 ### Configure general settings
 
-1. Sign in to [Intune](https://go.microsoft.com/fwlink/?linkid=2090973).
+1. Sign in to [Intune](https://endpoint.microsoft.com)).
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/education-settings-configure-ios.md:83]:
```suggestion
1. Sign in to [Intune](https://endpoint.microsoft.com).
```

**Comment by PRMerger12** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please review and signoff this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**